aloha-cli/src/test/scala/com/eharmony/aloha/cli/ModelTypesTest.scala

```diff
@@ -28,7 +28,8 @@ class ModelTypesTest {
       "ModelDecisionTree",
       "Regression",
       "Segmentation",
-      "VwJNI"
+      "VwJNI",
+      "multilabel-sparse"
```

Comment:
Why is the casing different here?

Reply:
Yeah.  I want opinions on the name here.  I am neither set on, nor particularly enamored with the name name `"multilabel-sparse"`.  Suggestions?